### PR TITLE
ens, installation chapters: general improvements

### PR DESCRIPTION
**Pull request by nonsense**

### CODE REVIEW

File: contents/installation.rst [hunk 1]
```diff
@@ -5,15 +5,31 @@ Installation and Updates
...
+.. code-block:: none
+
+  sudo add-apt-repository -y ppa:ethereum/ethereum
+
+After that you can install the stable version of Swarm:
+
+F. code-block:: none
+
+  sudo apt-get update
+  sudo apt-get install ethereum-swarm
```
Comment (justelad) [inline contents/installation.rst]: is this the renamed package name?

Comment (nonsense) [reply contents/installation.rst]: yes, we have a PR open on `go-ethereum` side, which is mostly approved.

Comment (justelad) [reply contents/installation.rst]: 👍

Comment (nonsense) [reply contents/installation.rst]: @homotopycolimit you're right. I will run through the commands as soon as we merge the `ethereum-swarm` PR and we get the first release out, and will update docs accordingly.

**Pull request merged**